formspree/forms/views.py

```diff
@@ -373,6 +374,18 @@ def unblock_email(email):
         return render_template('forms/unblock_email.html', email=email), 200
 
 
+def unconfirm_form(form_id, digest):
+    '''
+    We send a digest as the List-Unsubscribe header on every submission.
+    Here we get that digest and handle the unconfirmation request.
+    '''
+    form = Form.query.get(form_id)
+    if form.unconfirm_with_digest(digest):
+        return '', 200
```

Comment:
Why is this returning nothing, shouldn't we display a confirmation page?

Reply:
This is intended to be used only as the "One-Click" action specified at https://www.ietf.org/rfc/rfc2369.txt (email clients are supposed to send the POST themselves when the user clicks on a magic "unsubscribe" link), this URL should never be clicked by anyone.

If we want to show a clickable and visible "unconfirm" link, [it's much more complicated](https://github.com/formspree/formspree/pull/122).